Console app ls command/Console app ls command/showFiles.swift

```diff
@@ -0,0 +1,68 @@
+//
+//  showFiles.swift
+//  Console app ls command
+//
+//  Created by Viktor Rezvantsev on 02.03.2020.
+//  Copyright © 2020 Viktor Rezvantsev. All rights reserved.
+//
+
+import Foundation
+
+struct FileDescription: Codable {
+    var referenceCount: Int
+    var ownerAccountName: String
+    var fileSize: UInt64
+    var modificationDate: Date
+}
+
+let fm = FileManager.default
+let path = Bundle.main.resourcePath!
```

Comment:
The idea was to print contents of current directory. I think `Bundle.main.resourcePath` is not the most reliable means to get it. I'd suggest using FileManager.default.currentDirectoryPath